idunn/places/pj_poi.py

```diff
@@ -419,3 +420,48 @@ def get_raw_grades(self):
 
     def get_reviews_url(self):
         return self.get_source_url() + "#ancreBlocAvis"
+
+    def get_booking_url(self):
+        return next(
+            (
+                link.url
```

Comment:
These new functions should use `resolve_url()` to build a URL that will be resolved on server-side, similarly to `get_website()`.